x64/Release/obs-websocket_autogen/obs-websocket_autogen.log

```diff
@@ -0,0 +1 @@
+﻿  Automatic MOC and UIC for target obs-websocket
```

Comment:
And another file that slipped in :smile:

Reply:
Add "/obs-websocket.dir/" to .gitignore

Reply:
Don't forget to remove the actual file! I can't merge these unwanted files.